<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeElementWidget</class>
 <widget class="QWidget" name="RecipeElementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">RecipeElement</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="m_btnIcon">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DSGUI::DSToolListView" name="m_pToolListView">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
       <item>
        <widget class="QStackedWidget" name="funcStackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="mainApp::RecipeListWidget" name="m_pRecipeWidget"/>
         <widget class="mainApp::ElementListWidget" name="m_pElementWidget"/>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="DSGUI::DSPushButton" name="m_btnFront">
         <property name="text">
          <string>上一页</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="DSGUI::DSPushButton" name="m_btnExport">
         <property name="text">
          <string>文案导出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DSGUI::DSPushButton</class>
   <extends>QPushButton</extends>
   <header>PushButton/PushButton.h</header>
  </customwidget>
  <customwidget>
   <class>DSGUI::DSToolListView</class>
   <extends>QFrame</extends>
   <header>ToolListView/ToolListView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mainApp::ElementListWidget</class>
   <extends>QWidget</extends>
   <header>ElementListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mainApp::RecipeListWidget</class>
   <extends>QWidget</extends>
   <header>RecipeListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
